docs/function-schemas.md

```diff
@@ -368,17 +373,18 @@ Example of a multi-arity function with instrumentation scopes and custom reporti
       ([x y] (* x y)))))
 
 (multi-arity-pow 4)
-; =prints=> :malli.core/invalid-output {:output [:int {:max 6}], :value 16, :args [4], :schema [:=> [:cat :int] [:int {:max 6}]]}
-; => 16
+;; =stdout=> :malli.core/invalid-output {:output [:int {:max 6}], :value 16, :args [4], :schema [:=> [:cat :int] [:int {:max 6}]]}
```

Comment:
This is an example of leveraging test-doc-blocks' capability of verifying what it printed in stdout.